shared/src/clients/mail-client.class/index.ts

```diff
@@ -8,6 +8,10 @@ const CONFIGURATION_SET_HEADER = 'X-SES-CONFIGURATION-SET' // Case sensitive
 
 export * from './interfaces'
 
+export type SendEmailOpts = {
+  extraSmtpHeaders: Record<string, any>
```

Comment:
I don't really understand what's happening with this SMTP headers thing!

Reply:
Can elaborate more?

Reply:
Oh I just don't really understand this part of the PR haha — maybe can find some time to talk through for my understanding? 🙏

Reply:
Yea can u elaborate what it is that u dont understand?